drivers/bridge/bridge.go

```diff
@@ -360,17 +374,38 @@ func (d *driver) configure(option map[string]interface{}) error {
 				logrus.Warnf("Running modprobe bridge br_netfilter failed with message: %s, error: %v", out, err)
 			}
 		}
-		removeIPChains()
-		natChain, filterChain, isolationChain1, isolationChain2, err = setupIPChains(config)
+
+		removeIPChains(iptables.IPv4)
```

Comment:
shouldn't we remove/setup either ipv4 or ipv6 chains in this function ?

Reply:
The old patch was build in a way that it only enables the IPv6 handling if also the IPv4 handling is enabled.
For sure it is an option to enable it independent.

Reply:
resorted the EnableIP6Tables usage in `driver.configure`